test/suites/debug_compilation_dir.bash

```diff
@@ -0,0 +1,49 @@
+SUITE_debug_compilation_dir_PROBE() {
```

Comment:
Thanks for adding tests!

Reply:
No problem. Note that I mostly copied it from test/suites/debug_prefix_map.bash.